tuna/example/example_tables.py

```diff
@@ -49,11 +50,13 @@ class JobEnum(enum.Enum):
 
 class Job(BASE):
   """Represents class for job table"""
-  __tablename__ = "job"
-  __table_args__ = (UniqueConstraint('reason', 'session', name="uq_idx"),)
+  __tablename__: str = "job"
+  __table_args__: Tuple[UniqueConstraint] = (UniqueConstraint('reason',
```

Comment:
I'm not sure what the right typing for this is, but something seems off.

Reply:
@cderb I'd recommend https://github.com/ROCmSoftwarePlatform/MITuna/pull/912#discussion_r1224266350